### PR TITLE
OSDOCS-4582

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-4582](https://issues.redhat.com//browse/OSDOCS-4582)

Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OSDOCS-4582

Link to docs preview:
ROSA: https://56783--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jneczypor** [comment]
Hello @yuwang-RH! Can you please give this PR a review when you have time? Thank you!

**Review by arendej — APPROVED** [review]
LGTM, thanks.

File: modules/osd-aws-privatelink-firewall-prerequisites.adoc [hunk 1]
```diff
@@ -72,6 +72,22 @@ This section provides the necessary details that enable you to control egress tr
...
+|443, 80
+|The `sso.redhat.com` site redirects through `www.redhat.com`.
+
+|`aws.amazon.com`
+|443
+|The `iam.amazonaws.com` and `sts.amazonaws.com` sites redirect through `aws.amazon.com`.
+
+|`catalog.redhat.com`
+|443
+|The `registry.access.redhat.com` and `http://registry.redhat.io` sites redirect through `catalog.redhat.com`.
```
Comment (eohartman) [inline modules/osd-aws-privatelink-firewall-prerequisites.adoc:90]: For catalog.redhat.com, the website address should be https:, not http:. Everything else looks great!

**Review by eohartman — CHANGES_REQUESTED** [review]

**Review by arendej — APPROVED** [review]
lgtm

**Review by kelbrown20 — COMMENTED** [review]
LGTM as well

**Pull request merged**